indigo/bulk_creator.py

```diff
@@ -358,7 +358,10 @@ def validate_row(self, row):
 
         row = SpreadsheetRow(form.cleaned_data, errors)
         # has the work (implicitly) commenced?
-        row.commenced = bool(row.commencement_date or row.commenced_by)
+        # if the commencement date has an error, the row won't have the attribute
+        row.commenced = bool(
+            (row.commencement_date if hasattr(row, 'commencement_date') else False) or
```

Comment:
would `getattr(row, 'commencement_date', None)` be clearer?

Reply:
Much clearer, thank you!